kit/src/components/message/mod.rs

```diff
@@ -88,11 +87,15 @@ pub struct Props<'a> {
 }
 
 fn wrap_links_with_a_tags(text: &str) -> String {
-    let re = Regex::new(r#"(?i)\b((?:https?://|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))"#).unwrap();
+    let re = regex::Regex::new(r#"(?i)\b((?:(?:https?://|www\.)[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))"#).unwrap();
 
     re.replace_all(text, |caps: &regex::Captures| {
         let url = caps.get(0).unwrap().as_str();
-        format!("<a href=\"{}\">{}</a>", url, url)
+        if url.starts_with("www.") {
+            format!("<a href=\"https://{}\">{}</a>", url, url)
```

Comment:
Just for clarification, are we to assume every url will be https? Not that this is blocking or anything but just curious (although it is clear from the regex that it is)

Reply:
initially i added http but hogan suggested to force the https instead 🔨  what do you think we should improve? 💥